tools/azure-sdk-tools/ci_tools/environment_exclusions.py

```diff
@@ -33,6 +33,7 @@
     "azure-media-analytics-edge",
     "azure-media-videoanalyzer-edge",
     "azure-template",
+    "azure-purview-datamap",
```

Comment:
Please don't add your package in the list 
![image](https://github.com/Azure/azure-sdk-for-python/assets/70930885/0b0eb234-8e4d-469b-9ed1-26b09700f0b8)

Reply:
I am following this TSG to fix the Pylint/Mypy/Pyright failures. Please help suggest how should we resolve the failures. Thanks.
![image](https://github.com/Azure/azure-sdk-for-python/assets/78060542/a5d7f962-2572-4a69-bcb6-2555c54c9a73)

Reply:
You could follow https://github.com/Azure/azure-sdk-for-python/blob/main/doc/eng_sys_checks.md#the-pyprojecttoml to fix it for now (e.g. https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/purview/azure-purview-sharing/pyproject.toml)